### ISSUE TITLE
Change Msg2MimeTransform using ruby-msg library

### ISSUE THREAD

**Issue by phgross** [4teamwork/opengever.core#750]
The [`Msg2MimeTransform`](https://github.com/4teamwork/opengever.core/blob/master/opengever/base/transforms/msg2mime.py#L10) should be reimplemented using the [ruby-msg](https://github.com/aquasync/ruby-msg) library instead of the [Perl script](https://github.com/4teamwork/opengever.core/blob/master/opengever/base/transforms/msg2mime.pl)

The ruby-msg library would be much easier for the deployment.

**Comment by lukasgraf** [comment]
**Note**: `ruby-msg` currently only works with Ruby 1.8 (see https://github.com/aquasync/ruby-msg/pull/5 and https://github.com/aquasync/ruby-msg/issues/9).

**Comment by jevgtv** [comment]
Yes, it  works with Ruby 1.8 only, but recent Ubuntu and linux mint releases forse Ruby 1,9. What to do? How to overcome that?

**Comment by Rotonen** [comment]
We're still on the Perl script and both distro packages and CPAN are working out for us just fine?
